client/packages/common/src/ui/layout/tables/components/Cells/DotCell/DotCell.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { CellProps } from '../../../columns';
+import { FiberManualRecordIcon } from '@openmsupply-client/common';
+import { RecordWithId } from '@common/types';
+import { useBufferState } from '@common/hooks';
+
+export const DotCell = <T extends RecordWithId>({
+  rowData,
+  column,
+}: CellProps<T>): React.ReactElement<CellProps<T>> => {
+  const [buffer] = useBufferState(column.accessor({ rowData }));
```

Comment:
I doubt that the buffer is required. That is used in the checkbox cell to buffer user input changes (keeping props in sync).
In this case, it's readonly and you just need to get a boolean value out of the accessor.

I wonder if `DotCell` is expressive enough. Does it say 'will show a dot if the provided value is true and a blank otherwise' clearly?
I'm thinking it's ok - mainly due to a lack of good alternative suggestions!

Reply:
maybe DotCheckCell (extending the existing CheckCell name)